crates/iota/src/iota_commands.rs

```diff
@@ -988,6 +994,16 @@ async fn genesis(
         .map(SnapshotSource::S3);
     genesis_conf.migration_sources = local_snapshots.chain(remote_snapshots).collect();
 
+    // A delegator must be supplied when migration snapshots are provided.
+    if !genesis_conf.migration_sources.is_empty() {
+        if let Some(delegator) = delegator {
+            // Add a delegator account to the genesis.
+            genesis_conf = genesis_conf.add_delegator(delegator);
+        } else {
+            bail!("A delegator must be supplied when migration snapshots are provided.");
```

Comment:
Typically we use lower-case first letters in error messages. Please review other error messages in the patch as well.
```suggestion
            bail!("a delegator must be supplied when migration snapshots are provided.");
```

Reply:
Done